### DISCUSSION TITLE
Can you host experiments on a cloud rather than local server?

### DISCUSSION THREAD

**Discussion by Ricci-H** [jspsych/jsPsych#2818] [General]
Hi, 
This is probably a silly question... I've used jsPsych to code and run a couple of behavioural experiments. The task was hosted on a university server, and the data was saved there too. No problems. 
I was wondering if you can run experiments if they are hosted on a cloud? I've tried (e.g. from  Sharepoint), but the task doesn't run. It just wants to open the html file as a document. 
Thanks,
Ricci

**Comment by jodeleeuw** [comment]
Hi @Ricci-H,

Yes, you can host experiments on a cloud provider, but probably not one that is designed for sharing document files. You'll need something that is a web server. I would look for services that specifically mention hosting websites and web applications. If you want to save the data from the experiment, then you'll still need some kind of server-side script to handle writing data files.

**Comment by Ricci-H** [comment]
Thanks for the quick reply Josh. I'll take a closer look.
